.gitignore

```diff
@@ -1,9 +1,11 @@
 .idea
-scripts/bin
 node_modules
+scripts/bin
+scripts/src/**/*.d.ts
```

Comment:
these shouldn't be here.
your ide/env should not produce these files to begin with.
this is true to `.d.ts`, `.js` and `.js.map` files but only in the `scripts/src` directory, the config for the tests is different